api/types.proto

```diff
@@ -459,12 +459,26 @@ enum TaskState {
 	// Remove this message when the states are deemed perfect.
 }
 
+// State of Networks attached to a container
+message ContainerNetworkStatus {
+	// Network is the Network ID
+	string network = 1;
+
+	// Interface is the device name as seen within the container
+	string interface = 2;
+
+	// Address is the IP address in the form ipv4:XXX or ipv6:YYY
+	string address = 3;
```

Comment:
Wondering if it would make sense to break apart the protocol and address into separate fields.

If this format is borrowed from CNI and is well-defined there, this is probably fine, but in general I like to avoid microformats.